lib/plugins/bayesian-optimization/bloc/bayesian_optimization_bloc.dart

```diff
@@ -156,8 +156,8 @@ class BayesianOptimizationBloc extends BiocentralBloc<BayesianOptimizationEvent,
           'target_interval_lb': event.targetRangeMin?.toString() ?? event.targetValue?.toString() ?? '',
           'target_interval_ub': event.targetRangeMax?.toString() ?? event.targetValue?.toString() ?? '',
           'value_preference': switch (event.optimizationType.toString()) {
-            'Maximize' => 'maximise',
-            'Minimize' => 'minimise',
+            'Maximize' => 'maximize',
+            'Minimize' => 'minimize',
             _ => 'neutral',
```

Comment:
Could probably be replaced by 
`'value_preference': event.optimizationType?.toLowerCase() ?? 'neutral'`